doc/user_management.md

```diff
@@ -1,27 +1,123 @@
+
+
+## Manage users credentials
+
+ 
+***IMPORTANT: *** By default, only admin(root) OS user can change credentials for TC users. If there is a specific need for a new OS user to access and modify the configurations, more information can be found [here](./advancedConfiguration/manage-os-users.md).
+
+ECC `users.properties` can be found in next folders: [***ecc_resources_provider (ECC Provider)***](../ecc_resources_provider/users.properties)  and  [***ecc_resources_consumer (ECC consumer)***](../ecc_resources_consumer/users.properties).
+
+[here](../ecc_resources_consumer/users.properties)
+
+DataApp `users.properties` can be found in next folder: [***be-dataapp_resouces***](../be-dataapp_resources/users.properties)
+
+
+### Usernames naming convention
+
+If your company doesn't have established naming convention, you can follow these basic guidelines:
+
+* Uniqueness: Ensure each username is unique.
+* Character Set: Stick to letters and numbers.
+* Length: Keep it short, around 3-20 characters.
+* No Personal Info: Avoid using personal data like full names or birthdays.
+* Case Insensitivity: Make usernames case-insensitive for user convenience.
+* Reserved Keywords: Don't allow reserved or restricted words.
+
+
+### Add new user
+
+As mentioned earlier `user.list` property is a list containing usernames, with each username separated by a comma (,) but without any spaces. It is essential to individually assign each user a specific password, as demonstrated in the example. These passwords must be encoded using BCrypt. Getting passwords can be done via following endpoint:
 
 ```
-application.security.password=$2a$10$MQ5grDaIqDpBjMlG78PFduv.AMRe9cs0CNm/V4cgUubrqdGTFCH3m
+/api/password/{new_password}
+```
+
+Using this endpoint, it is guaranteed that the password strength rules configured in the `application.properties` file will be enforced.
+
+Bare in mind that this endpoint is password protected, and you will have to provide existing credentials in order for TRUE Connector to generate new hash that matches with the value passed in URL, so the general advice is to keep `apiUser` as a kind of administrator account. Once new hash is returned, you can modify properties file and set new password for specific user.
+
+Example:
 
 ```
+# List of users
+users.list=newUser
 
-while apiUser is present in ecc property file. 
+# Credentials for each user
+# encoded - passwordNewUser123
+newUser.password=$2a$12$v7/AKsx5KTNJpwOg9yRRe.h6jP80gc03umqqN6aMAQtvVEWmpIqna
+```
+
+After the files are modified, it is necessary to restart container in order to apply changes.
+
+
+### Modify existing user
+
+Same rules applies to user credentials modification, what it needs to be updated is username in the `user-list`, and if password need to be updated, the same endpoint  should be used. 
```

Comment:
Should we mention that you can change username, password or both?
Should we add in user-list some oldUsername1,oldUsername2 just to show difference that we are adding new user next to existing ones?